### ISSUE TITLE
[Package Request] Modern alternatives to common shell commands

### ISSUE THREAD

**Issue by fredrike** [SynoCommunity/spksrc#5104]
### Software Name

gdu (du), duf (df)

### Brief Description

There is a good write up on alternatives to the classics `du` and `df` here: https://itsfoss.com/legacy-linux-commands-alternatives/

My idea is to extend the `synocli-file` package with the following:

- [x]  bat
- [ ] gdu
- [x] exa
- [ ] duf

### Website

https://itsfoss.com/legacy-linux-commands-alternatives/

### Documentation

- gdu https://github.com/dundee/gdu
- duf https://github.com/muesli/duf

### Build/Installation documentation

- bat https://github.com/sharkdp/bat#from-source
- gdu https://github.com/dundee/gdu
- exa https://the.exa.website/install/source
- duf https://github.com/muesli/duf#from-source

### Source code

- bat https://github.com/sharkdp/bat
- gdu https://github.com/dundee/gdu
- exa https://github.com/ogham/exa
- duf https://github.com/muesli/duf

### OpenSource License

MIT

**Comment by publicarray** [comment]
Hi thanks for the suggestions. `bat` and `exa` are already included: https://github.com/SynoCommunity/spksrc/blob/f34c825d99a09221c46add3f260cb8d3b63e2bbf/spk/synocli-file/Makefile#L14

Currently bat on my system is not build correctly @hgy59:

```
emory allocation of 7881122326526296064 bytes failed
Aborted (core dumped)
```

**Comment by publicarray** [comment]
Never mind 0the newly published one: Version 2.5-13 works. Thanks, @hgy59
